universal-application-tool-0.0.1/app/views/style/BaseStyles.java

```diff
@@ -11,25 +11,28 @@
   // CiviForm color classes
   /////////////////////////////////////////////////////////////////////////////////////////////////
 
+  public static final String BG_CIVIFORM_WHITE = "bg-civiformWhite";
+
   public static final String BG_SEATTLE_BLUE = "bg-seattleBlue";
-  public static final String BG_CIVIFORM_BLUE_60 = "bg-civiformBlue-60";
-  public static final String BG_CIVIFORM_BLUE_80 = "bg-civiformBlue-80";
-  public static final String BG_CIVIFORM_BLUE_100 = "bg-civiformBlue-100";
+  public static final String TEXT_SEATTLE_BLUE = "text-seattleBlue";
```

Comment:
question (non-blocking): Out of curiosity, is there a plan to move these to be more general, i.e. another city can use their own color scheme?

Reply:
Yes, I really hope so! It's something we haven't talked about yet, but need to.

Reply:
I figured I'd just add the Seattle color scheme in this hard-coded way for now for the pilot. I'd love to somehow make this configurable.